version_helpers.py

```diff
@@ -0,0 +1,72 @@
+''' Distutils / setuptools helpers for versioning
+
+This code started life in the nibabel package as nisexts/sexts.py
```

Comment:
Honestly I would recommend getting rid of this code now, and preferring `versioneer` which gives version numbers from the git tag.

But - copying this function allows dipy to install without doing a previous and temporary install both of nibabel and numpy (because nibabel depends on numpy).   With the copy we can let pip sort out the nibabel and numpy dependencies by a standard install.